output/cloud/expv2/flush.go

```diff
@@ -37,9 +41,23 @@ func (f *metricsFlusher) Flush(ctx context.Context) error {
 	msb := newMetricSetBuilder(f.referenceID, f.aggregationPeriodInSeconds)
 	for i := 0; i < len(buckets); i++ {
 		msb.addTimeBucket(&buckets[i])
```

Comment:
```suggestion
		msb.addTimeBucket(buckets[i])
```
nit: I am not really certain why you need a pointer here :shrug: